CMakeLists.txt

```diff
@@ -68,6 +68,7 @@ if(UNIX)
   option(ENABLE_INTERNAL_GTEST "Enable internal gtest?" OFF)
   option(ENABLE_INTERNAL_UDFREAD "Enable internal udfread?" OFF)
   option(ENABLE_INTERNAL_SPDLOG "Enable internal spdlog?" OFF)
+  option(ENABLE_INTERNAL_DATE "Enable internal date library?" ON)
```

Comment:
Is this meant to be default ON whereas most everything else is default OFF?

Reply:
Well, we can safely make it opt-in just like the rest of dependencies. We just need to build it along with other dependencies like spdlog or gtest. Debian uses ON here with component tarball fed to `FindDate.cmake` via `DATE_URL` option.

Reply:
Yes should probably default to OFF as most platforms will build the depends version instead of the cmake external project version.